src/Traits/InertiaResponseTrait.php

```diff
@@ -30,10 +33,40 @@ trait InertiaResponseTrait
         'JsonViewClass' => \CakeDC\Inertia\View\InertiaJsonView::class,
     ];
 
-    public function buildPaginationLinks(PaginatedInterface $paging)
+    public function parseRelated(array $data, string $model): array
+    {
+        foreach ($data as $key => $val) {
+            if (is_array($val)) {
+                if (array_key_exists('_ids', $val)) {
+                    $newItems = [];
+                    foreach ($val as $item) {
+                        foreach ($item as $value) {
+                            $newItems[] = $value['id'];
+                        }
+                    }
+                    $data[$key]['_ids'] = $newItems;
+                }
+            }
+        }
+
+        $db = ConnectionManager::get('default');
+        $collection = $db->getSchemaCollection();
+        $tableSchema = $collection->describe($model);
+        foreach ($tableSchema->columns() as $column) {
+            if ($tableSchema->getColumnType($column) == 'timestampfractional' && $data[$column] !== null) {
+                $data[$column] = Date::parseDate($data[$column], 'YYYY-MM-dd');
```

Comment:
formatting could be different in specific projects?

Reply:
could it be true, I pass the format as a parameter?